### PR TITLE
Enable suggestion functionality on GitHub

### PR DESCRIPTION
**Pull request by EGiataganas**

### CODE REVIEW

File: danger-kotlin-library/src/main/kotlin/com/danger/dangerkotlin/DangerDSL.kt [hunk 1]
```diff
@@ -11,4 +11,12 @@ data class DangerDSL(
         @SerializedName("bitbucket_server")
         val bitBucketServer: BitBucketServer,
         val git: Git
-)
\ No newline at end of file
+) {
+        fun onGithub() : Boolean {
```
Comment (gianluz) [inline danger-kotlin-library/src/main/kotlin/com/danger/dangerkotlin/DangerDSL.kt]:
```suggestion
        fun onGithub() = github != null
```

Comment (gianluz) [reply danger-kotlin-library/src/main/kotlin/com/danger/dangerkotlin/DangerDSL.kt]: this function can stay in the same line without body

File: danger-kotlin-library/src/main/kotlin/com/danger/dangerkotlin/DangerDSL.kt [hunk 2]
```diff
@@ -11,4 +11,12 @@ data class DangerDSL(
...
+                return github != null
+        }
+
+        fun onBitBucketServer() : Boolean {
```
Comment (gianluz) [inline danger-kotlin-library/src/main/kotlin/com/danger/dangerkotlin/DangerDSL.kt]: same here ;)

**Comment by f-meloni** [comment]
merge on green

**Pull request merged**